### ISSUE TITLE
Question about pre-trained detr for vcoco

### ISSUE THREAD

**Issue by kaijieshi7** [SherlockHolmes221/DOQ#10]
Hello. I want to know if the pre-trained vcoco model you provided was train like this? 
https://github.com/hitachi-rd-cv/qpic/issues/4#issuecomment-801738132

**Comment by SherlockHolmes221** [comment]
no, We initialize the network with the parameters of DETR  trained on MS-COCO database
